book.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'rental'
+class Book
+  attr_accessor :author, :title, :rental
+
+  def initialize(title, author, rental)
+    @title = title
+    @author = author
+    @rental = rental
```

Comment:
- [ ] Kindly make sure the rentals are an empty array instead as the previous reviewer suggested

```suggestion
    @rental = []
```